peachjam/templates/peachjam/judgment_detail.html

```diff
@@ -119,28 +119,65 @@
 {% endblock %}
 {% block extra-metadata %}
   <dl class="document-metadata-list d-md-flex">
-    {% if document.flynote %}
-      <dt>
-        {% trans 'Flynote' %}
-      </dt>
-      <dd>
-        {{ document.flynote|safe }}
-      </dd>
-    {% endif %}
     {% if document.case_summary %}
       <dt>
-        {% trans 'Case summary' %}
+        {% trans 'Short summary' %}
       </dt>
       <dd>
-        {{ document.case_summary|safe }}
+        {% if document.blurb %}<div class="mb-3">{{ document.blurb|default_if_none:"" }}</div>{% endif %}
+        <details>
+          <summary class="mb-1">
+            {% trans 'Read full summary' %}
+          </summary>
+          {% if perms.peachjam.can_view_document_summary %}
+            {% if document.case_summary %}
+              <div class="mb-2">
+                <h6 class="fw-bold">{% trans 'Case summary' %}</h6>
+                {{ document.case_summary|safe }}
+              </div>
+            {% endif %}
+            {% if document.issues %}
+              <div class="mb-2">
+                <h6 class="mb-1 fw-bold">{% trans 'Issues' %}</h6>
+                <ol>
+                  {% for issue in document.issues %}<li>{{ issue }}</li>{% endfor %}
+                </ol>
+              </div>
+            {% endif %}
+            {% if document.held %}
+              <div class="mb-2">
+                <h6 class="mb-1 fw-bold">{% trans 'Held' %}</h6>
+                <ol>
+                  {% for held in document.held %}<li>{{ held }}</li>{% endfor %}
+                </ol>
+              </div>
+            {% endif %}
+            {% if document.order %}
```

Comment:
the order provided by the summariser service isn't being stored in `Judgment.generate_summary`